snuba/snapshots/postgres_snapshot.py

```diff
@@ -120,12 +121,44 @@ def load(cls, product: str, path: str) -> PostgresSnapshot:
     def get_descriptor(self) -> PostgresSnapshotDescriptor:
         return self.__descriptor
 
+    def __table_iterable(self, csv_table) -> Generator[Mapping[str, Any], None, None]:
+        for r in csv_table:
+            yield r
```

Comment:
Is this a placeholder for something that was planned to be filled in here? This doesn't otherwise seem any different than just iterating `csv_table` itself.

Reply:
I created this function to be able to return a generator easily at line 160. I am not sure if this is possible by inlining a lambda there.

Reply:
I guess does this actually need to be a generator and not just an iterator?

Reply:
turns out it did not. I removed it.